src/main/resources/view/TransactionListCard.fxml

```diff
@@ -9,7 +9,8 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<?import javafx.scene.layout.RowConstraints?>
```

Comment:
Move this import up with the other imports, and leave a line after the last import.